config/routes.rb

```diff
@@ -4,11 +4,16 @@
     root to: "devise/registrations#edit"
   end
 
+  use_doorkeeper do
+    skip_controllers :tokens, :authorizations, :token_info, :authorized_applications
+  end
+
   scope :api do
     scope :v1 do
       use_doorkeeper do
         controllers tokens: 'api/v1/tokens'
-        skip_controllers :applications, :authorizations, :token_info
+
+        skip_controllers :applications, :authorizations, :token_info, :authorized_applications
```

Comment:
1. Why is `skip_controllers` duplicated 🤔 ?
2. We must keep `applications` controller to manage the list of application, no?

Reply:
I wanna mount `applications` route to `/` instead of inside `/api/v1`, so I need to define `skip_controllers` twice.

Then we can use the following link to manage the OAuth applications:
`https://survey-api.nimblehq.co/oauth/applications`

I think it will be better than `https://survey-api.nimblehq.co/api/v1/oauth/applications`

Reply:
I added some comments in code, please help me check